media/js/base/sentry.es6.js

```diff
@@ -22,7 +22,8 @@ if (
             dsn: sentryDsn,
             sampleRate: 0.1,
             ignoreErrors: [
-                'https://plugin.ucads.ucweb.com/api/flow/',
+                "SecurityError: Failed to execute 'open' on 'XMLHttpRequest'/", // issue 10683
```

Comment:
Do we need the slash inside quotes?
```suggestion
                "SecurityError: Failed to execute 'open' on 'XMLHttpRequest'", // issue 10683
```